service/internal/handler/count.go

```diff
@@ -0,0 +1,42 @@
+package handler
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/kimbellG/packet-filter/service/pkg/coderror"
+	"github.com/kimbellG/packet-filter/service/pkg/codes"
+)
+
+func (h *Handler) GetCount(w http.ResponseWriter, r *http.Request) {
+	HTTPHandlerWithError(h.getCount).Serve(w, r)
```

Comment:
Very weird way to use middleware. Why do we need GetCount if you're just going to wrap h.getCount with a middleware? 
1. I would rename HTTPHandlerWithError to something like ErrorMiddleware.
2. I would rename getCount to GetCount and when I register the endpoints I would just do **r.HandleFunc("/count", ErrorMiddleware(h.GetCount))**